Retrying failed tasks
---------------------

An automatic task might also fail for an external reason that you can resolve.
A :ref:`google-drive` task will fail if the configured Google account doesn’t have permission to write to the selected Google Drive folder, for example.
You can manually retry some failed tasks, after resolving the external issue, so that task completes successfully and case execution continues.

.. figure:: /_static/images/tasks/retry.png

   An option to retry a task that could succeed on the next try

To retry a failed task, use the *Retry task* button to the right of the task name in the case’s task list.
